WcaOnRails/db/structure.sql

```diff
@@ -1366,6 +1085,7 @@ CREATE TABLE `votes` (
 /*!50001 SET character_set_results     = utf8 */;
 /*!50001 SET collation_connection      = utf8_general_ci */;
 /*!50001 CREATE ALGORITHM=UNDEFINED */
+/*!50013 DEFINER=`root`@`localhost` SQL SECURITY DEFINER */
```

Comment:
`SELECT VERSION()` results in `5.7.19-0ubuntu0.16.04.1` for me! Also note that the view that is generated for you doesn't make sense either, I mean the fact that all fields have the type of `tinyint`. I think it depends on the MySQL version. Do you know more about this security definer thing?